src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -33,12 +35,14 @@ public class AdminController : ApiController
         private readonly WebScriptHostManager _scriptHostManager;
         private readonly WebHostSettings _webHostSettings;
         private readonly TraceWriter _traceWriter;
+        private readonly ILogger _logger;
 
-        public AdminController(WebScriptHostManager scriptHostManager, WebHostSettings webHostSettings, TraceWriter traceWriter)
+        public AdminController(WebScriptHostManager scriptHostManager, WebHostSettings webHostSettings, TraceWriter traceWriter, ILoggerFactory loggerFactory)
         {
             _scriptHostManager = scriptHostManager;
             _webHostSettings = webHostSettings;
             _traceWriter = traceWriter.WithSource($"{ScriptConstants.TraceSourceHostAdmin}.Api");
+            _logger = loggerFactory?.CreateLogger("Host.Admin.Api");
```

Comment:
It looks like we're having the same duplicate logging pattern as in the other review. I thought you said you had a solution for that? It's a maintenance issue to have to remember to log double everywhere and ensure that the logs are the same